string command
string data
---
string output
